### DISCUSSION TITLE
v2.0.0

### DISCUSSION THREAD

**Discussion by nickfloyd** [octokit/octokit.net#2528] [Announcements]
## Advisories and Breaking Changes

In this release, we are dropping support for .NET 4.6.  This needs to be done to move forward with the framework and innovate using the new APIs and assemblies.

Highlights from the [issue](https://github.com/octokit/octokit.net/issues/2500) description:
> .NET Framework 4.6 (and 4.6.1) went out of support on April 26, 2022. Continuing to publish Octokit for this target adds a support burden.

> .NET standard 2.0 covers .NET Framework 4.6.2 already.

> Supporting a single target framework, .NET standard 2.0, would simplify maintenance of this project while supporting all target frameworks currently supported upstream.

For reference, these are the supported versions going forward

.NET implementation | Version support
-- | --
.NET and .NET Core | 2.0, 2.1, 2.2, 3.0, 3.1, 5.0, 6.0
.NET Framework  | 4.6.1 2, 4.6.2, 4.7, 4.7.1, 4.7.2, 4.8

For more information see #2521 via [@nickfloyd](https://github.com/nickfloyd)

## Release Notes

**Fixes**

- Fixes issues with two-factor authentication overload and certain endpoints - [#2524](https://github.com/octokit/octokit.net/pull/2524) via [@JonruAlveus](https://github.com/JonruAlveus)

**Housekeeping**

- Bumps xunit from 2.4.1 to 2.4.2 - [#2518](https://github.com/octokit/octokit.net/pull/2518) via [@dependabot[bot]](https://github.com/apps/dependabot)

**Documentation Updates**

- Updates deprecation warning from subscribers_count to watchers_count - [#2519](https://github.com/octokit/octokit.net/pull/2519) via [@Brend-Smits](https://github.com/Brend-Smits)
- Updates code docs for SetRequestTimeout to more appropriately reflect what the method does. - [#2520](https://github.com/octokit/octokit.net/pull/2520) via [@notauserx](https://github.com/notauserx)

<hr /><em>This discussion was created from the release <a href='https://github.com/octokit/octokit.net/releases/tag/v2.0.0'>v2.0.0</a>.</em>

**Comment by JamieMagee** [comment]
Almost a decade to get to v1.0.0. Just over 3 weeks to get from v1.0.0 to v2.0.0 😂
